lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -104,8 +105,9 @@ export class PublicClientApplication {
      * Use when initiating the login process by redirecting the user's browser to the authorization endpoint.
      * @param {@link (AuthenticationParameters:type)}
      */
-    loginRedirect(request: msalAuth.AuthenticationParameters): msalAuth.TokenResponse {
-        throw new Error("Method not implemented.");
+    async loginRedirect(request: AuthenticationParameters): Promise<TokenResponse> {
+        console.log(await this.authModule.createLoginUrl(request));
```

Comment:
Does this `console.log()` need to be here?

Reply:
Only here for testing, this will be removed going forward